app/library/ZapActor.scala

```diff
@@ -34,10 +34,7 @@ import org.apache.http.client.entity.UrlEncodedFormEntity
 import org.apache.http.client.methods.HttpPost
 import org.apache.http.impl.client.HttpClientBuilder
 import org.apache.http.message.BasicNameValuePair
-import play.api.libs.json.Json
-import play.api.libs.ws.WS
 import play.libs.Akka
-import play.api.Play.current
```

Comment:
? not sure you want to remove the static 'current' here

Reply:
IntelliJ will mark it as "unused" but it's important for Akka context for ZapActor